CHANGELOG.md

```diff
@@ -916,165 +901,158 @@ In MLflow Projects:
+- [Models] It is now possible to pass extra arguments to `mlflow.keras.load_model` that will be passed through to `keras.load_model`. (#1330, @yorickvP)
+- [Serving] For better performance, switch to `gunicorn` for serving Python models. This does not change the user interface. (#1322, @tomasatdatabricks)
 - [Deployment] For SageMaker, use the uniquely-generated model name as the S3 bucket prefix instead of requiring one. (#1183, @dbczumar)
-- [REST API] Add support for API paths without the ``preview`` component. The ``preview`` paths will be deprecated in a future version of MLflow. (#1236, @mparkhe)
+- [REST API] Add support for API paths without the `preview` component. The `preview` paths will be deprecated in a future version of MLflow. (#1236, @mparkhe)
 
 Bug fixes and documentation updates
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 - [Tracking] Log metric timestamps in milliseconds by default (#1177, @smurching; #1333, @dbczumar)
-- [Tracking] Fix bug when deserializing integer experiment ID for runs in ``SQLAlchemyStore`` (#1167, @smurching)
+- [Tracking] Fix bug when deserializing integer experiment ID for runs in `SQLAlchemyStore` (#1167, @smurching)
 - [Tracking] Ensure unique constraint names in MLflow tracking database (#1292, @smurching)
 - [Tracking] Fix base64 encoding for basic auth in R tracking client (#1126, @freefrag)
-- [Tracking] Correctly handle ``file:`` URIs for the ``-—backend-store-uri`` option in ``mlflow server`` and ``mlflow ui`` CLI commands (#1171, @eedeleon, @tomasatdatabricks)
+- [Tracking] Correctly handle `file:` URIs for the `-—backend-store-uri` option in `mlflow server` and `mlflow ui` CLI commands (#1171, @eedeleon, @tomasatdatabricks)
 - [Artifacts] Update artifact repository download methods to return absolute paths (#1179, @dbczumar)
 - [Artifacts] Make FileStore respect the default artifact location (#1332, @dbczumar)
-- [Artifacts] Fix ``log_artifact`` failures due to existing directory on FTP server (#1327, @kafendt)
+- [Artifacts] Fix `log_artifact` failures due to existing directory on FTP server (#1327, @kafendt)
 - [Artifacts] Fix GCS artifact logging of subdirectories (#1285, @jason-huling)
-- [Projects] Fix bug not sharing ``SQLite`` database file with Docker container (#1347, @tomasatdatabricks; #1375, @aarondav)
-- [Java] Mark ``sendPost`` and ``sendGet`` as experimental (#1186, @aarondav)
-- [Python][CLI] Mark ``azureml.build_image`` as experimental (#1222, #1233 @sueann)
+- [Projects] Fix bug not sharing `SQLite` database file with Docker container (#1347, @tomasatdatabricks; #1375, @aarondav)
+- [Java] Mark `sendPost` and `sendGet` as experimental (#1186, @aarondav)
+- [Python][cli] Mark `azureml.build_image` as experimental (#1222, #1233 @sueann)
```

Comment:
prettier automatically lower-cases link keys:
https://github.com/prettier/prettier/issues/7118

Reply:
One way to prevent this is to wrapping with backticks:

```
`[Python][CLI]` Mark ...
```

Reply:
so let's change it back to [CLI] ?

Reply:
@WeichenXu123 If we use `[Python][CLI]`, we can't apply auto-formatting with prettier.

Reply:
got it. I think using either ``[Python][CLI]`` or `[Python/CLI]` is fine.

Reply:
+1 to the hybridized [Python/<PascalCase/camelcase>] as needed so that link references are consistent and legible.